cime_config/testdefs/ExpectedTestFails.xml

```diff
@@ -233,17 +219,17 @@
     </phase>
   </test>
 
-  <test name="PEM_D_Ld15.5x5_amazon.I2000Clm50FatesRs.cheyenne_gnu.clm-FatesColdSeedDisp">
+  <test name="PEM_D_Ld15.5x5_amazon.I2000Clm50FatesRs.derecho_gnu.clm-FatesColdSeedDisp">
     <phase name="COMPARE_base_modpes">
       <status>FAIL</status>
       <issue>FATES#1089</issue>
     </phase>
   </test>
 
-  <test name="PEM_D_Ld15.5x5_amazon.I2000Clm50FatesRs.derecho_gnu.clm-FatesColdSeedDisp">
```

Comment:
This actually should be added back in, just for the f10_f10 grid, so I'll do that.